app/assets/javascripts/plot/highchartsops.coffee

```diff
@@ -92,7 +92,7 @@ class window.HighchartsOps extends PlotOps
     super(resize, reset, registerPen, resetPen, addPoint, updatePenMode, updatePenColor)
     @_chart              = Highcharts.chart(elemID, {})
     @_penNameToSeriesNum = {}
-
+    @_chart.options.exporting.buttons.contextButton.menuItems.pop()
```

Comment:
Yes, I've made such a mess of this PR, I apologize. I'm going to rebase to a single commit.